repo/packages/S/spark/0/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "postInstallNotes": "The Apache Spark DCOS Service has been successfully installed!\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK",
+  "scm": "https://github.com/apache/spark.git",
+  "images": {
+    "icon-medium": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-large.png",
+    "icon-small": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-small.png"
+  },
+  "maintainer": "marco.capuccini@it.uu.se",
+  "preInstallNotes": "Note that the Apache Spark DCOS Service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of two nodes with at least 2 CPU and 2GB of RAM available for the Spark Service and running a Spark job.\nNote: The Spark CLI may take up to 5min to download depending on your connection.",
```

Comment:
do you mind just removing the dcos references here and in config.json? You could just replace it with something like `Apache Spark Mesos Framework'